tgui/packages/tgui-panel/stat/reducer.js

```diff
@@ -65,5 +66,18 @@ export const statReducer = (state = initialState, action) => {
       dead_popup: false,
     };
   }
+  if (type === 'stat/alertBr') {
+    return {
+      ...state,
+      alert_popup: null,
```

Comment:
I think the loop that sends the alert message to people should just send the BR alert to admins and never send them the round end alert at all.